_includes/section-sidebar-menu.html

```diff
@@ -5,6 +5,7 @@
     <h2>Menu</h2>
     <p>{{ locale.project.description }}</p>
   </header>
+  <span>Elixir {{site.elixir.version}} Erlang/OTP {{site.erlang.OTP}} [erts-{{site.erlang.erts}}]</span>
```

Comment:
We need something to separate Elixir version from Erlang/OTP other than just a space.
maybe a `-`

Reply:
![image](https://user-images.githubusercontent.com/73386/39283767-16642b5a-48cd-11e8-947f-e0e405d92cad.png)